hyperspy/_signals/eds.py

```diff
@@ -980,7 +980,7 @@ def _add_vertical_lines_groups(self, position, **kwargs):
         """
         per_xray = len(position[0])
         colors = itertools.cycle(np.sort(
-            plt.rcParams['axes.color_cycle'] * per_xray))
```

Comment:
Wouldn't it be better to support old and new versions checking the mpl version? e.g. `if StrictVersion(matplotlib.__version__) < x:`